scripts/format.py

```diff
@@ -120,51 +121,33 @@
 force = False
 
 
-def print_usage():
-    print("Usage: python scripts/format.py [revision|--all] [--check|--fix] [--force]")
-    print(
-        "   [revision]     is an optional revision number, all files that changed since that revision will be formatted (default=HEAD)"
-    )
-    print("                  if [revision] is set to --all, all files will be formatted")
-    print("   --check only prints differences, --fix also fixes the files (--check is default)")
+parser = argparse.ArgumentParser(prog="python scripts/format.py", description='Format source directory files')
+parser.add_argument('--revision', type=str, default='HEAD', help='Revision to format from (default: HEAD)')
```

Comment:
```py
parser.add_argument('revision', nargs='?', default='HEAD', help='Git revision to check (default: HEAD)')
```
leaving off the `--` makes it a positional argument, keeping the original behavior.